dotcom-rendering/src/lib/contributions.ts

```diff
@@ -14,14 +14,9 @@ import type { DCRTagPageType } from '../types/tagPage';
 // User Attributes API cookies (created on sign-in)
 export const HIDE_SUPPORT_MESSAGING_COOKIE = 'gu_hide_support_messaging';
 export const RECURRING_CONTRIBUTOR_COOKIE = 'gu_recurring_contributor';
-export const ONE_OFF_CONTRIBUTION_DATE_COOKIE = 'gu_one_off_contribution_date';
 export const OPT_OUT_OF_ARTICLE_COUNT_COOKIE = 'gu_article_count_opt_out';
 
-// Support Frontend cookies (created when a contribution is made)
-export const SUPPORT_RECURRING_CONTRIBUTOR_MONTHLY_COOKIE =
```

Comment:
It's safe to remove these because support-frontend sets the `gu_hide_support_messaging` cookie after checkout - is that right?

Reply:
Yes exactly 👍